### PR TITLE
Task main page

### PR DESCRIPTION
**Pull request by ajhswan**

### CODE REVIEW

File: collectionApp_2020-02-04.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# ************************************************************
```
Comment (robings) [inline collectionApp_2020-02-04.sql]: delete the old database (or move it somewhere outside of the folders git is checking)

File: index.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$receipt = displayData($result);
+?>
+
+<!DOCTYPE html>
+
+<html lang='en-gb'>
+
+<head>
+    <title>Collection App</title>
+    <link rel='stylesheet' type='text/css' href='normalize.css'/>
```
Comment (robings) [inline index.php:23]: Do you need to include `<meta name='viewport' content='width=device-width' />` in the head.

**Review by robings — CHANGES_REQUESTED** [review]
looks good, just a couple of minor comments

File: collectionApp_2020-02-04.sql (hunk 1 above)
Comment (ajhswan) [reply collectionApp_2020-02-04.sql]: thanks deleted

File: index.php (hunk 2 above)
Comment (ajhswan) [reply index.php:23]: added thanks

**Pull request merged**